plugins/vendor/zoom/lib/model/ZoomVendorIntegration.php

```diff
@@ -185,7 +185,7 @@ public function shouldExcludeUserRecordingsIngest($userId)
 		{
 			return false;
 		}
-		$userGroupsArray = KuserKgroupPeer::retrievePgroupIdsByKuserIds($userId);
+		$userGroupsArray = KuserKgroupPeer::retrievePgroupIdsByPuserIds($userId);
```

Comment:
extract the kuserId from the partnerId + puserId and use the previous function since puserId is not a key on kuser_kgroup table and its less efficient